WeatherPortal/WeatherApp/WeatherApp/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for WeatherApp project.
+
+Generated by 'django-admin startproject' using Django 4.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-6-z8i1$hlnjq0&b$@vrgae!p%uze9jov51pc@xe$v4-)3i=8-l'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
No tutaj chcielibyśmy takich rzeczy nie mieć na repo, mamy korzystać z zmiennych środowiskowych

Reply:
Po prostu usunąć to ?